Micronaut  Example Main class was not found, perhaps you should add a `mainClass` configuration to jib

<!-- Thanks for filing an issue! Please use this template to provide more context for the issue . -->

**Description of the issue**:
Micronaut example not compiling

**Expected behavior**:
create docker image

**Steps to reproduce**:  

./gradlew jibDockerBuild

**Environment**: <!-- OS, Maven, and other relevant environment information -->

Linux, gralde, JDK 11

 
**`jib-gradle-plugin` Configuration**: <!-- Delete this section if not used -->
```groovy
plugins {
    id 'application'
    id 'groovy'
    id 'io.spring.dependency-management' version '1.0.6.RELEASE'
    id 'net.ltgt.apt-idea' version '0.18'
    id 'com.google.cloud.tools.jib' version '1.0.0'
}

version '0.1'
group 'example.micronaut-jib'
sourceCompatibility = 1.8

repositories {
    mavenCentral()
    maven { url 'https://jcenter.bintray.com' }
}

dependencyManagement {
    imports {
        mavenBom 'io.micronaut:micronaut-bom:1.0.3'
    }
}

dependencies {
    annotationProcessor 'io.micronaut:micronaut-inject-java'
    compile 'io.micronaut:micronaut-http-client'
    compile 'io.micronaut:micronaut-http-server-netty'
    compile 'io.micronaut:micronaut-runtime-groovy'
    compile 'io.micronaut:micronaut-inject'
    compile 'io.micronaut:micronaut-runtime'
    compileOnly 'io.micronaut:micronaut-inject-groovy'
    compileOnly 'io.micronaut:micronaut-inject-java'
    runtime 'ch.qos.logback:logback-classic:1.2.3'
    testCompile 'io.micronaut:micronaut-inject-groovy'
    testCompile('org.spockframework:spock-core:1.2-groovy-2.5') {
        exclude group: 'org.codehaus.groovy', module: 'groovy-all'
    }
    testCompile 'io.micronaut:micronaut-inject-java'
}

compileJava.options.compilerArgs += '-parameters'
compileTestJava.options.compilerArgs += '-parameters'
run.main = 'example.micronaut.Application'


```

**Log output**: <!-- If applicable, provide relevant log output -->

./gradlew jibDockerBuild  --image=test
> Task :jibDockerBuild FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jibDockerBuild'.
> com.google.cloud.tools.jib.plugins.common.MainClassInferenceException: Main class was not found, perhaps you should add a `mainClass` configuration to jib

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.


**Additional Information**: <!-- Any additional information that may be helpful -->


<!-- Thanks for contributing! -->

Hmm this seems to be deeper than jib:
```sh
$ ./gradlew build
> Task :startScripts FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':startScripts'.
> No value has been specified for property 'mainClassName'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
4 actionable tasks: 1 executed, 3 up-to-date
```

--- a/examples/micronaut/build.gradle
+++ b/examples/micronaut/build.gradle
@@ -3,7 +3,7 @@ plugins {
     id 'groovy'
     id 'io.spring.dependency-management' version '1.0.6.RELEASE'
     id 'net.ltgt.apt-idea' version '0.18'
-    id 'com.google.cloud.tools.jib' version '1.0.0'
+    id 'com.google.cloud.tools.jib' version '1.0.1'
 }
 
 version '0.1'
@@ -43,6 +43,3 @@ mainClassName = 'example.micronaut.Application'
 
 /** CHANGE THIS TO THE IMAGE YOU WANT TO PUSH TO */
 // jib.to.image = 'gcr.io/PROJECT_ID/micronaut-jib'
-
-// not needed once https://github.com/GoogleContainerTools/jib/issues/1501 is fixed
-jib.container.mainClass = mainClassName

--- a/examples/micronaut/build.gradle
+++ b/examples/micronaut/build.gradle
@@ -43,4 +43,6 @@ mainClassName = 'example.micronaut.Application'
 
 /** CHANGE THIS TO THE IMAGE YOU WANT TO PUSH TO */
 // jib.to.image = 'gcr.io/PROJECT_ID/micronaut-jib'
+
+// not needed once https://github.com/GoogleContainerTools/jib/issues/1501 is fixed
 jib.container.mainClass = mainClassName

--- a/examples/micronaut/build.gradle
+++ b/examples/micronaut/build.gradle
@@ -39,7 +39,8 @@ dependencies {
 
 compileJava.options.compilerArgs += '-parameters'
 compileTestJava.options.compilerArgs += '-parameters'
-run.main = 'example.micronaut.Application'
+mainClassName = 'example.micronaut.Application'
 
 /** CHANGE THIS TO THE IMAGE YOU WANT TO PUSH TO */
 // jib.to.image = 'gcr.io/PROJECT_ID/micronaut-jib'
+jib.container.mainClass = mainClassName

